src/styles/SearchBar.css

```
@@ -0,0 +1,28 @@
+.input-search-bar {
+    width: 668px;
+    height: 36px;
+    border-radius: 99px;
+    border: none;
+    background: url(../utils/images/lupa.svg) 15px no-repeat #EDEDF0;
```

Acho que em vez de usar as cores como # (hastag), o que acha de usar com o rgb()? O ESlint não trabalha muito bem com as cores em # 😄 
```suggestion
    background: url(../utils/images/lupa.svg) 15px no-repeat rgb( 237, 237, 40 );
```

---

Blz, ajustando.